modular_skyrat/modules/xenos_skyrat_redo/code/larva.dm

```diff
@@ -0,0 +1,66 @@
+/// SKYRAT MODULE SKYRAT_XENO_REDO
+
+/datum/action/cooldown/alien/larva_evolve/Activate(atom/target)
+	var/mob/living/carbon/alien/larva/larva = owner
+	var/static/list/caste_options
+	if(!caste_options)
+		caste_options = list()
+
+		// This --can probably-- (will not) be genericized in the future.
+		var/datum/radial_menu_choice/runner = new()
+		runner.name = "Runner"
+		runner.image  = image(icon = 'modular_skyrat/modules/xenos_skyrat_redo/icons/xeno_actions.dmi', icon_state = "preview_runner")
+		runner.info = span_info("Runners are the most agile caste, the short stature of running on all fours \
+		gives them great speed, the ability to dodge projectiles, and allows them to tackle while holding throw and clicking. \
+		Eventually, runners can evolve onwards into the fearsome ravager, should the hive permit it.")
+
+		caste_options["Runner"] = runner
```

Comment:
I feel like this can be made into a proc, so as to avoid so much repetition.

Thank you :)

Reply:
I mean maaaaaybe